sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -93,6 +94,13 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
       );
     }
 
+    if (application.isArchived) {
```

Comment:
mm, you can add this logic in the query where condition and adjust line 92 message

Reply:
If the logic is added to the query it will throw not found exception instead of unprocessibleEntity.

Reply:
Like-wise the logic inside inside the function is meant only for completed applications but we throw error104 to return Http422(UnProcessible entity). 

Btw. I checked the controller. It is throwing  422 on all the cases. At least the error description is more precise.
![image](https://user-images.githubusercontent.com/54600590/171748246-36e24c94-e8cc-4c4c-8c25-82fa1ad0e0b3.png)

Reply:
@ann-aot @dheepak-aot I personally had done on both ways and received PR comments on both occasions 😄 
I believe that both are valid, as explained below.
For the way that Dheepaks is saying unless there is a performance hit, we can do it when we need to be more specific and provide better individual messages for the errors.
For the way that Ann is describing we can compensate by including in the error message further explanation.

Reply:
if it's a requirement in the story, then Iam Good with it. My point is  in a normal scenario, as per my understanding, I think this scenario won't happen, because the active applications page will only have the `isArchived=false` applications, and only these applications will have the `report a change ` option unless someone tries to override the  `isArchived=true` application number from the URL (In that case error in line 92 or adjusting msg should be fine, IMO)

Reply:
@guru-aot @henryoforeh-aot  please share your thoughts  as well

Reply:
@dheepak-aot as I mentioned above, this point is part of the story, so, don't have any problem
![image](https://user-images.githubusercontent.com/77353155/171925282-ccd0990c-4ecd-4ec7-8488-d3b91fd0a05e.png), the comment was my opinion 😄 . if you want to talk about it we can have a different call😄

Reply:
In my opinion I would leave it where it is.

Reply:
I am good here @ann-aot  😊